src/connectors/answers/connectAnswers.ts

```diff
@@ -89,8 +90,16 @@ export type AnswersConnectorParams = {
   extraParameters?: FindAnswersOptions;
 };
 
+export type AnswersWidgetDescription = {
+  $$type: 'ais.answers';
+  renderState: AnswersRenderState;
+  indexRenderState: {
+    answers: WidgetRenderState<AnswersRenderState, AnswersConnectorParams>;
```

Comment:
```suggestion
    answers: AnswersRenderState;
```

I wish it could just be this, and then augmented with the parameters when you use it (eg. in Widget['getRenderState']), but I can't get it to work, so for now, the index render state only types the connector params, not the connector & widget params. I think this is a reasonable tradeoff for now, considering the use cases for reading the widget params from index render state are likely slim (and can use an `as any`)

Reply:
Looks good to me. We could improve this part later if we find a solution.